helloworldmtls/README.md

```diff
@@ -8,3 +8,7 @@ go run ./helloworldmtls/worker -target-host my.namespace.tmprl.cloud:7233 -names
 ```
 go run ./helloworldmtls/starter -target-host my.namespace.tmprl.cloud:7233 -namespace my.namespace -client-cert path/to/cert.pem -client-key path/to/key.pem
 ```
+
+If the server uses self-signed certificates, pass one of the following two options when starting the worker or the example above:
```

Comment:
```suggestion
If the server uses self-signed certificates and does not have the SAN set to the actual host, pass one of the following two options when starting the worker or the example above:
```

Just having self-signed is not a problem. The problem is they don't have the host you are trying to connect to in the SAN of the certificate. In theory we should discourage SNI override and skip-verify, but the options are fine to have in this sample.

Reply:
Thank you for the suggestion – you're absolutely right. 🎉